src/node_crypto.cc

```diff
@@ -3320,6 +3320,9 @@ void Hmac::HmacDigest(const FunctionCallbackInfo<Value>& args) {
 
   bool r = hmac->HmacDigest(&md_value, &md_len);
   if (!r) {
+    if ( md_value != nullptr ) {
+        delete[] md_value;
+    }
```

Comment:
Not an actual change, but to make an analysis tool happy about a false positive.
1. `md_value` is initially set to `nullptr` a pair of lines above.
2. `HmacDigest(&md_value, &md_len)` is defined above, and it could only return `false` at the start, and that means that it did nothing. In all other cases, it returns `true`.
3. The above means that if `HmacDigest(&md_value, &md_len)` returned `false`, then `md_value` is guaranteed to be `nullptr` and this check is noop.

The compiler is smart enough to check this, so the check will also probably be optimized out by the compiler. This could only serve for aesthetic purposes and making your (non ideal) analysis tool happy.